src/resolver.js

```diff
@@ -24,99 +23,132 @@ const resolvers = {
         const user = await db.collection("users").findOne({id});
         return !error ? user : {...user, id: 1}
     },
-    createUser: async ({id, firstName, password}, context) => {
+    createUser: async ({user: {id, name, password}}, context) => {
         const {db} = await context();
-        //хэширование пароля
         const hashPassword = await bcrypt.hash(password, 12);
-        const newUser = {id, firstName, password: hashPassword};
+        const newUser = {id, name, password: hashPassword};
+        console.log('New user: ', newUser)
         db.collection("users").insertOne(newUser);
```

Comment:
Why you didn't use mongoose models?

Reply:
Sadly, I haven't seen any examples project using both technologies: "graphql" & "mongoose models".. I thought it a little wrong;)